AnimLib.csproj

```diff
@@ -1,23 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
-  <Import Project="..\..\references\tModLoader.targets" />
+  <Import Project="..\tModLoader.targets" />
   <PropertyGroup>
     <AssemblyName>AnimLib</AssemblyName>
-    <TargetFramework>net45</TargetFramework>
-    <PlatformTarget>x86</PlatformTarget>
-    <LangVersion>7.3</LangVersion>
-    <Description>Terraria library mod for enhanced animations and ability management.</Description>
-    <RepositoryUrl>https://github.com/TwiliChaos/AnimLib/</RepositoryUrl>
-    <RepositoryType>git</RepositoryType>
+    <TargetFramework>net6.0</TargetFramework>
+    <PlatformTarget>AnyCPU</PlatformTarget>
+    <LangVersion>latest</LangVersion>
+	<Description>Terraria library mod for enhanced animations and ability management. (1.4 port)</Description>
```

Comment:
Could you remove (1.4 port) here and change the repo url back?

Reply:
Ok, done